armi/utils/dynamicImporter.py

```diff
@@ -24,7 +24,9 @@
 def importEntirePackage(module):
     """Load every module in a package
 
-    NOTE: this method may only work for a flat directory?
+    Notes
+    -----
+    This method may only work for a flat directory?
```

Comment:
Not a huge fan of having a comment visible in the API that has a question mark on it.

I guess whatever, and not at all related to the current PR intent, but for how long are we gonna kick this can down the road?

Reply:
That's a fair point.  I didn't put the question mark there, so I just didn't see it.

I'll just test that statement to see if it's true, and adjust the note accordingly.

Reply:
Oh, yeah, there is no reason for that "?" at all, that little method is dead simple and only works on flat directories. I cleaned up the comment.